### PR TITLE
Add gst-libav

### PR DESCRIPTION
**Pull request by ryonakano**
See https://github.com/otsaloma/gaupol/blob/master/flatpak/io.otsaloma.gaupol.yml and  https://github.com/ryonakano/reco/commit/fa42e07334641f6c0ac4cfd1f76680e8fa3d9226

### CODE REVIEW

**Comment by dar5hak** [comment]
Thanks a lot @ryonakano!

Just one question: is 1.16 an LTS (or otherwise important) version?

I am trying to only keep 1 or 2 most recent stable versions, and [seems like](https://gstreamer.freedesktop.org/src/gst-libav/) 1.19.1 is the latest. But if you think 1.16 is a useful version to have, we can have both.

If not, we can just link to the [latest tarball](https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.19.1.tar.xz) and its [sha256](https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.19.1.tar.xz.sha256sum).

**Comment by ryonakano** [comment]
It seems like we need to use the version between gstreamer and gst-libav and I chose 1.16 because elementary Sdk comes with gstreamer 1.16.3 and gst-libav fails to build on it:

Screenshot from 2021-07-30 21-53-11

But gst-libav 1.19 should work if you uses gstreamer 1.19, so maybe we can add both 1.19 and 1.16 in this json.

File: public/modules.json [hunk 1]
```diff
@@ -226,5 +226,27 @@
...
+          "sources": [
+            {
+              "type": "git",
+              "url": "git://anongit.freedesktop.org/gstreamer/gst-libav",
+              "tag": "1.16"
+            }
+          ]
+        },
+        "finish-args": "--env=GST_PLUGIN_PATH_1_0=/app/lib/gstreamer-1.0"
+      }
```
Comment (dar5hak) [inline public/modules.json]: That's a great point. How about this?

```suggestion
      {
        "version": "1.19.1",
        "module": {
          "name": "gst-libav",
          "buildsystem": "autotools",
          "config-opts": "--with-system-libav=no",
          "sources": [
            {
              "type": "archive",
              "url": "https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.19.1.tar.xz",
              "sha256": "7a36c466a0942667f5be5d427d0840d1b81c187a4b93a73dcf10e53bf9cf196c"
            }
          ]
        },
        "finish-args": "--env=GST_PLUGIN_PATH_1_0=/app/lib/gstreamer-1.0"
      },
      {
        "version": "1.16.3",
        "module": {
          "name": "gst-libav",
          "buildsystem": "autotools",
          "config-opts": "--with-system-libav=no",
          "sources": [
            {
              "type": "archive",
              "url": "https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.16.3.tar.xz",
              "sha256": "d10c5eb1a00a91de97c85c0956c663aa6e99d268195cdec4534c179b831538ec"
            }
          ]
        },
        "finish-args": "--env=GST_PLUGIN_PATH_1_0=/app/lib/gstreamer-1.0"
      }
```

Comment (ryonakano) [reply public/modules.json]: Yes, looks fine. Between 1.16 and 1.19 they seemed to switch to meson so we need to take care of that though.

File: public/modules.json [hunk 2]
```diff
@@ -235,8 +235,7 @@
         "version": "1.19.1",
         "module": {
           "name": "gst-libav",
-          "buildsystem": "autotools",
-          "config-opts": "--with-system-libav=no",
+          "buildsystem": "meson",
```
Comment (dar5hak) [inline public/modules.json:238]: Good catch! 😅

**Comment by dar5hak** [comment]
Dōmo arigatō!

**Pull request merged**